SnakeGame/SnakeGame/Utils/StandardMessages.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace SnakeGame.Utils
+{
+    public class StandardMessages
```

Comment:
We are not only storing the message content but we also writing them better name for the class